sig-platform/community-donated-plugins.md

```diff
@@ -0,0 +1,13 @@
+# Community Donated Plugins
+
+The [`spinnaker-plugins`](https://github.com/spinnaker-plugins) GitHub organization is a holding repository for unofficial, community-contributed Spinnaker plugins.
```

Comment:
Is there any plan to make plugins stored here somehow "easy to find", e.g. to index them somewhere, or perhaps even to make them easily installable in Spinnaker itself?

Reply:
Yes! https://github.com/spinnaker/plugins is the draft for a plugin repository. It's similar in concept to maven central, except that we'll be offering just pointers to artifact storage, rather than storing the artifacts themselves. The community has expressed interest in creating a marketplace at some point. This repository concept makes plugins easy to install, there's some docs on that repo of what it looks like (basically, if you don't pre-download the plugins, the services will download and side-load the plugins at startup - the `sha512sum` is required).

Reply:
Ack, thanks for explaining! Is that something we could mention here, or is it not sufficiently ready for that?

Reply:
It's still something we're working through - probably not for another quarter.

Reply:
Clear, thanks!